cmd/livepeer/livepeer.go

```diff
@@ -183,19 +191,23 @@ func main() {
 	} else if transcoder {
 		n.NodeType = core.TranscoderNode
 		glog.Info("***Livepeer is in transcoder mode ***")
-		server.RunTranscoder(n, *orchAddr)
+		server.RunTranscoder(n, orchAddresses[0])
 		return
 	} else {
 		n.NodeType = core.BroadcasterNode
 	}
 
 	if n.NodeType == core.BroadcasterNode {
-		if *orchAddr == "" {
+		if orchAddresses[0] == "" {
 			glog.Info("No orchestrator specified; transcoding will not happen")
 		} else {
-			n.OrchestratorSelector = discovery.NewOffchainOrchestrator(n, *orchAddr)
-			if n.OrchestratorSelector == nil {
-				return
+			if len(orchAddresses) > 0 {
+				n.OrchestratorSelector = discovery.NewOffchainOrchestrator(n, orchAddresses[0])
```

Comment:
This probably just hasn't been implemented yet, but we're going to pass in the entire `orchAddresses` list here, right?

Reply:
That's right. I'm working on that now :)